### ISSUE TITLE
Add admin interface to create records

### ISSUE THREAD

**Issue by javpet** [AgileVentures/sing_for_needs#84]
## Expected Behavior

Admin interface to create records we can play around in the environment.
https://github.com/mojotech/torch

## Current Behavior

We don't have an admin yet.

## Possible Solution

Adding torch for our existing models.

**Comment by okothkongo** [comment]
do really need torch considering this an API, and we rarely dealing with backend and not worrying much about frontend and from my  understanding of torch it is majorly a template generator

**Comment by javpet** [comment]
Yeah it seems Torch gives a simple web GUI to basically to CRUD operations for us in mainly the development phase similarly as Active Admin in Rails. Do you have another suggestion @okothkongo1? Something better we can use for the same purpose?
